Indexing/SubstitutionTree_FastGen.cpp

```diff
@@ -477,7 +477,7 @@ bool SubstitutionTree::FastGeneralizationsIterator::findNextLeaf()
   CALL("SubstitutionTree::FastGeneralizationsIterator::findNextLeaf");
 
   Node* curr;
-  bool sibilingsRemain;
+  bool sibilingsRemain = false;
```

Comment:
I was unable to persuade GCC's control flow analyzer that these are in fact assigned before use, and I'm not completely sure myself because of the nested-loop-goto thing below. Set-to-zero is more or less free on modern hardware, save us some non-deterministic debugging later.